fishnet.py

```diff
@@ -1656,22 +1734,19 @@ def update_available():
                      __version__, latest_version)
         return True
 
+def display_config(args, conf, update=True):
+    '''
+    Display args and conf settings
 
-def cmd_run(args):
-    conf = load_conf(args)
-
-    if args.auto_update:
-        print()
-        print("### Updating ...")
-        print()
-        update_self()
+    This potentially updates StockfishCommand as a side-affect of how getting that command works.
+    '''
```

Comment:
Maybe the function should be renamed to something like `prepare_config()`, then?

Reply:
I broke this into two parts `update_config` and `display_config`